projects/crypto-currency/iluminados/app/src/components/Coinoftheday/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react'
+import './style.css'
+
+function CoinOfTheDay() {
+
+  const [list, setList] = useState([])
+  const API = process.env.REACT_APP_API_URL
+
+  useEffect(() => {
+    fetch(
+      `${API}coinoftheday`,
```

Comment:
É preciso ajustar no BE, nome de api deve ser separado por hífen

`coin-of-the-day`